incubator/hnc/internal/validators/object_test.go

```diff
@@ -308,6 +309,20 @@ func TestUserChanges(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "Deny creation of object with invalid select annotation",
```

Comment:
Should have an "allow" as well.

Also, how about the scenario where there are multiple selectors (eg `.../treeSelector` and `.../selector`) on the same object?

Reply:
Added "allow" test. I'll add the multiple selector case in the other PR (as /treeSelect is not validated here yet)